opencti-platform/opencti-front/src/private/components/common/form/mcas/CountryEnum.ts

```diff
@@ -0,0 +1,257 @@
+enum Countries {
```

Comment:
Instead of using an enum for Country fields, you should rather use the stix countries of the platform. You can create a new relationship between countries and threat actor individuals for the place of residence (ex: is-located-at), an other for the place of birth, etc.

Reply:
@Archidoit / @Bonsai8863  - We used an ENUMs for country - as it is very "static" and doesn't require the running of the OpenCTI seeding connector (external-import/opencti) to populate the system for support. Thus, can work on an instance that has not been "seeded" using this connector. This might be a bit of a re-engineering lift/effort to remap to country links/relationships that relies on the connector provided entities.  Would you consider this a requirement for initial merge/adoption - or could it be mapped as a tech-debt/backlog item with migration script to re-map - and the current approach merged as-is?

Reply:
@Archidoit These changes are addressed.
Added new relationships to replace:
- x_mcas_country_of_residence -> resides-in
- x_mcas_citizenship -> citizen-of
- x_mcas_place_of_birth -> born-in
User can add these relationships from a Threat Actor Individual to a Country under the knowledge tab.

Reply:
That's all good, thanks! StixCoreRelationships are adapted for these types of links, except for born-in. 'born-in' should rather be a ref relationship. Indeed stixCoreRelationships are adapted for relations that can change over time, have end dates etc. StixRefRelationships are more adapted for relations that don't change over time (like the place of birth).

Reply:
@Archidoit born-in is now a ref relationship. It can be set via the Threat Actor Individual create or edit panels.